src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/RemoteProjectSettingsUpdateRequest.java

```diff
@@ -0,0 +1,48 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto;
+
+import ca.corefacility.bioinformatics.irida.model.project.ProjectSyncFrequency;
+
+/**
+ * Used to handle requests from the UI to update settings for a remote sync'd project.
+ * The field is the attribute to be update, and the value is the value to set to that field.
+ */
+
+public class RemoteProjectSettingsUpdateRequest {
```

Comment:
There's enough of these `dto` classes to make a sub package for the Remote API stuff.  That package is getting messy.

Reply:
Sure, that makes sense. Updated in276dbcfde37eb7f873588b4ee2bb58652775aaf3